chapters/solutions.Rmd

```diff
@@ -206,15 +206,22 @@ The file should contain the following lines:
 
 ### Exercise \@ref(bash-basics-ex-pipe-construction) {-}
 
-```shell
-$ cut -d , -f 2 animals.txt | sort | uniq
-```
+`cut` selects substrings from a line by:
+
+-   breaking the string into pieces wherever it finds a separator (`-d ,`) and
+-   keeping one or more of the resulting fields (`-f 2`).
+
+Any single character can be used as a separator,
+but there is no way to escape characters:
+for example, if the string `a,"b,c",d` is split on commas,
+all three commas take effect.
 
 ### Exercise \@ref(bash-basics-ex-which-pipe) {-}
 
-Option 4. is the correct answer.
-If you have difficulty understanding why, try running the commands, or sub-sections of
-the pipelines (make sure you are in the `data-shell/data` directory).
+Option 4 is the correct answer.
+If you have difficulty understanding why,
+try running the commands or sub-sections of the pipelines
```

Comment:
Remove "or sub-sections" or clarify as "sections of commands between pipes" since we don't reference sub-sections in text anywhere